phase1/tf.jsonnet

```diff
@@ -75,37 +75,41 @@
     // ip addresses will cause all certs to be recreated tainting clusters using the
     // old certificates (i.e. causing those clusters to be recreated by terraform).
     cluster_tls(master_instance_names, master_instance_ips):: {
-      tls_private_key: {
-        [name]: pki.private_key
-        for name in ["root", "node", "master", "admin"]
-      },
-      tls_self_signed_cert: {
-        root: pki.tls_self_signed_cert("root"),
-      },
-      tls_cert_request: {
-        [name]: pki.tls_cert_request(name)
-        for name in ["node", "admin"]
-      } {
-        master: pki.tls_cert_request(
-          "master",
-          dns_names=master_instance_names + [
-            # master service dns names
-            "kubernetes",
-            "kubernetes.default",
-            "kubernetes.default.svc",
-            "kubernetes.default.svc.local",
-            "kubernetes.default.svc.local",
-          ],
-          ip_addresses=master_instance_ips + [
-            # master service ip, this depends on the cluster cidr
-            # so must be changed if/when we allow that to be configured
-            "10.0.0.1",
-          ],
-        ),
+      data: {
```

Comment:
I think this needs to be `data+` as well. Right now, your changes break Azure - I think it's because this masks the existing `data{}` that I have for the `template_file`s.

With your current commit (c209e9b), I get this:

```shell
+ terraform apply -state=./.tmp/terraform.tfstate .tmp
Error configuring: 2 error(s) occurred:

* azurerm_virtual_machine.master_vm: missing dependency: data.template_file.configure_vm
* azurerm_virtual_machine.node_vm: missing dependency: data.template_file.configure_vm
```